### ISSUE TITLE
relatedTarget for focus/blur events in jsdom

### ISSUE THREAD

**Issue by ph-fritsche** [testing-library/user-event#431]
## Feature request

It would be nice if `click` would trigger focus/blur events with `relatedTarget` property set to the previously active respectively the subsequently active element just [like it does in the browser](https://developer.mozilla.org/en-US/docs/Web/API/FocusEvent).

Current workaround is to trigger those per
```js
fireEvent.blur(element, {relatedTarget: nextActive})
fireEvent.focus(element, {relatedTarget: previousActive})
```

Is there any reason not to implement the same here?

https://github.com/testing-library/user-event/blob/6fa71631f8273c59023b59ba3befd204b8cd47d9/src/click.js#L64-L67

https://github.com/testing-library/user-event/blob/6fa71631f8273c59023b59ba3befd204b8cd47d9/src/blur.js#L9
https://github.com/testing-library/user-event/blob/6fa71631f8273c59023b59ba3befd204b8cd47d9/src/focus.js#L9

**Comment by kentcdodds** [comment]
Absolutely. If the browser does something, then we should do it as well. Would you be willing to [make a pull request](http://makeapullrequest.com) to implement this?

**Comment by krutoo** [comment]
Can we do same thing for dragLeave event?

**Comment by krutoo** [comment]
Can we do same behavior for drag events?
